api/src/main/java/eu/darkbot/api/game/galaxy/GalaxyGate.java

```diff
@@ -0,0 +1,65 @@
+package eu.darkbot.api.game.galaxy;
+
+import eu.darkbot.api.game.other.GameMap;
+import eu.darkbot.api.managers.StarSystemAPI;
+
+import java.util.Set;
+import java.util.stream.Collectors;
+
+/**
+ * Every GalaxyGate available in spinner.
+ */
+public enum GalaxyGate {
+    ALPHA  ("alpha",    1,  "α"),
+    BETA   ("beta",     2,  "β"),
+    GAMMA  ("gamma",    3,  "γ"),
+    DELTA  ("delta",    4,  "δ"),
+    EPSILON("epsilon",  5,  "ε"),
+    ZETA   ("zeta",     6,  "ζ"),
+    KAPPA  ("kappa",    7,  "κ"),
+    LAMBDA ("lambda",   8,  "λ"),
+    KRONOS ("kronos",   12, "Kronos"),
+    HADES  ("hades",    13, "Hades"),
+    KUIPER ("streuner", 19, "ς");
+
+    private final String name;
+    private final int id;
+    private final String mapSymbol;
+
+    GalaxyGate(String name, int id, String mapSymbol) {
+        this.name = name;
+        this.id = id;
+        this.mapSymbol = mapSymbol;
+    }
+
+    public String getParam() {
+        return "&gateID=" + getId() + "&" + getName() + "=1";
+    }
+
+    public String getIdParam() {
+        return "&gateID=" + getId();
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public int getId() {
+        return id;
+    }
+
+    public String getMapSymbol() {
+        return mapSymbol;
+    }
+
+    public boolean isInGate(StarSystemAPI starSystem) {
+        return starSystem.getCurrentMap().getName().contains(mapSymbol);
+    }
```

Comment:
```suggestion
    public boolean isInGate(GameMap map) {
        return map != null && map.getName().contains(mapSymbol);
    }
```
I think it makes way more sense to ask if a map is in gate, not if the star system is in the gate
It also adds more flexibility allowing any map to be checked